src/webcore/value.rs

```diff
@@ -1291,4 +1297,50 @@ mod tests {
         let a: Result< (), _ > = js! { return 1; }.try_into();
         assert!( a.is_err() );
     }
+
+    #[test]
+    fn reference_stable() {
+        js! { Module.__test = {}; }
+        let a = js! { return Module.__test; }.as_reference().unwrap().as_raw();
+        let b = js! { return Module.__test; }.as_reference().unwrap().as_raw();
+        assert_eq!(a, b);
+
+        js! { delete Module.__test; }
+        js! { Module.__test = {}; }
+        let c = js! { return Module.__test; }.as_reference().unwrap().as_raw();
+        assert_ne!(a, c);
```

Comment:
Is there any point in reassigning the `Module.__test` now? This might just as well be replaced with `js! { return {}; }`. (Since the fact that we're returning `Module.__test` doesn't matter; what matters here is that we `deleted`d the original `{}` and we returned a new `{}`.)

Reply:
I agree.